indico/modules/events/contributions/models/contributions.py

```diff
@@ -47,8 +47,25 @@ def _get_next_friendly_id(context):
     return increment_and_get(Event._last_friendly_contribution_id, Event.id == event_id)
 
 
+class CustomFieldsMixin(object):
+    """Methods to process custom field data."""
+
+    def set_custom_field(self, field_id, field_value):
+        fv = self.get_field_value(field_id, raw=True)
+        if not fv:
+            field_value_cls = type(self).field_values.prop.mapper.class_
+            fv = field_value_cls(contribution_field=self.event_new.get_contribution_field(field_id), data=field_value)
+            self.field_values.append(fv)
+        fv.data = field_value
+
+    def set_custom_fields(self, custom_fields_data):
```

Comment:
Or even better, set an attribute to the `Contribution` class: `custom_field_prefix = 'custom_'`. All of the models that need to support extra fields will eventually need a prefix for them so why not specifying that in the model?
Cause right now I have the hardcoded `'custom_'` string in `contributions.controllers.py`, `contributions.util.py` and the model.